config.js

```diff
@@ -88,6 +88,21 @@ module.exports = {
     // this will also cause sampling decisions made by other distributed components to be
     // ignored. This is useful for aggregating traces generated by different cloud platform
     // projects.
-    ignoreContextHeader: false
+    ignoreContextHeader: false,
+
+    // A path to a key file relative to the current working directory. If this
+    // field is set, the contents of the pointed file will be used for
+    // authentication instead of your application default credentials.
+    keyFile: null,
+
+    // An alias for keyFile. If both keyFile and keyFilename are set, the value
+    // of keyFilename will be ignored.
+    keyFilename: null,
```

Comment:
Was it a requirement to support this alias? I would prefer not to have duplicate config options unless there is a strong use case for different names?